_posts/2016-03-07-shaping-an-open-source-editorial-team.md

```diff
@@ -0,0 +1,74 @@
+---
+
+title: 'Shaping an Open Source editorial team
+'
+author: Jenn
+layout: blog-post
+comments: falst
+
+---
+
+#Shaping an open source editorial team
```

Comment:
oh also we had an issue with GitHub’s website generator for titles without a space between the `#` and the text. Can you add spaces to all the titles please?